src/components/SubHeader.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+import { SecondHeader } from '../styles/header';
```

Comment:
With the new package regading the relative path , shouldn't it be '/styles/' instead of '../styles/' ?